lib/robots/dor_repo/ocr/split_ocr_xml.rb

```diff
@@ -13,7 +13,7 @@ def initialize
         def perform_work
           base_output_path = Dor::TextExtraction::Ocr.new(cocina_object:).abbyy_output_path
           alto_path = File.join(base_output_path, "#{bare_druid}.xml")
-          return unless File.exist?(alto_path)
+          return LyberCore::ReturnState.new(status: :skipped, note: 'No full object XML file') unless File.exist?(alto_path)
 
```

Comment:
set the step to skipped with a note to make it clear nothing happened (this is for PDF/documents, which will not have full XML file)